plugins/generic/acron/AcronPlugin.inc.php

```diff
@@ -244,7 +244,11 @@ function shutdownFunction() {
 
 			// By updating the last run time as soon as feasible, we can minimize
 			// the race window. See bug #8737.
-			$updateResult = $taskDao->updateLastRunTime($className, time());
+			$tasksToRun = $this->_getTasksToRun();
```

Comment:
Just to double check: it is not important here to also actualize ` $this->_tasksToRun` because it is shutdown i.e. the last function?